dpnp/dpnp_iface_types.py

```diff
@@ -59,8 +59,14 @@
     "inf",
     "int",
     "int_",
+    "int8",
+    "int16",
     "int32",
     "int64",
+    "uint8",
```

Comment:
NumPy also provides aliases, like: `ubyte`, `ushort`, `ulong` and similar (full list here: `numpy/_core/numerictypes.pyi`)

Reply:
Added